spec/models/git_repository_spec.rb

```diff
@@ -188,5 +188,32 @@
       repo.refresh
       expect(repo.git_tags.collect(&:name)).to match_array(tag_list)
     end
+
+    context do
+      let(:dir_name) { repo.directory_name }
+      before do
+        FileUtils.mkdir_p dir_name
```

Comment:
@mkanoor Instead of creating this directory can we do these tests by stubbing the `FileUtils.rm_rf`?